src/main/java/org/sonatype/repository/chef/internal/security/ChefFormatSecurityContributor.java

```diff
@@ -10,19 +10,21 @@
  * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
  * Eclipse Foundation. All other trademarks are the property of their respective owners.
  */
-package org.sonatype.nexus.plugins.chef.internal.security;
+package org.sonatype.repository.chef.internal.security;
```

Comment:
In future can we keep the moves and new functionality in separate PRs please

Reply:
This was an outcome of using @doddi 's archetype, it sets the package name a bit wonky (I believe it should be what I've corrected to). I can easily do another PR for this to master and then merge in.

Reply:
Yep sounds good